### DISCUSSION TITLE
Is it expected the Nuget Embedded package for 6.0 includes lots of runtimes?

### DISCUSSION THREAD

**Discussion by kamranayub** [ravendb/ravendb#17696] [Q&A] [answered]
I was just cleaning up some space on my server and noticed the difference between 5.x and 6.x `RavenDB.Embedded` Nuget packages. The 6.0 packages a ton of runtimes which end up making it about twice the size.

Just wanted to make sure the team was aware, maybe it's expected.

![image](https://github.com/ravendb/ravendb/assets/563819/b8ee7385-b024-4b4b-a183-14fffdb7fc42)

**Comment by ppekrol** [comment, marked as answer]
hi @kamranayub 

We are aware of this. .NET runtime takes more space, but we also added a lot of additional libraries (e.g. for kafka support), which increased the size due to embedded native libs for a lot of distributions.

I wouldn't say it grew 7x though, but it doubled between 5.2 and 6.0. That is true.

| Version | NuGet (MB) | Uncompressed (MB)
| ------- | ------------------ | ------------------ |
| 6.0.0   | 89 | 201 |
| 5.4.111 | 80 | 185 |
| 5.3.108  | 43 | 95 |
| 5.2.116   | 37 | 86 |

**Reply by kamranayub** [reply]
🤦‍♂️ Sorry I was looking at the overall package folder, at 760mb not the individual versions. You're right of course 😅
